network/network_test.go

```diff
@@ -155,8 +158,8 @@ func TestNetwork(t *testing.T) {
 	confB := testConfig()
 	bootstrapPort := ts.RandInt32(9999) + 10000
 	confB.Listens = []string{
-		fmt.Sprintf("/ip4/0.0.0.0/tcp/%v", bootstrapPort),
-		fmt.Sprintf("/ip6/::/tcp/%v", bootstrapPort),
+		fmt.Sprintf("/ip4/127.0.0.1/tcp/%v", bootstrapPort),
```

Comment:
I think if we use 127.0.0.1 for tests it perfectly works on shell pipeline runners but in docker pipeline runners it would be better to use 0.0.0.0 which work's like 127.0.0.1 in shell runners.

Reply:
Same as above ☝️